docs/source/models.rst

```diff
@@ -266,52 +266,58 @@ For more info, see:
+These images can be deployed to Azure Kubernetes Service (AKS) and the Azure Container Instances (ACI)
+platform for real-time serving.
+
+* :py:func:`build_image <mlflow.azureml.build_image>` registers an MLflow model with an existing Azure ML
+  workspace and builds an Azure ML container image for deployment to AKS and ACI. The `Azure ML SDK`_ is
+  required in order to use this function.
 
-The :py:mod:`mlflow.azureml` module can export ``python_function`` models as Azure ML models and
-directly deploy and serve models on Azure ML, provided the environment has been set up.
+  .. _Azure ML SDK: https://docs.microsoft.com/en-us/python/api/overview/azure/ml/intro?view=azure-ml-py
 
-* :py:func:`export <mlflow.azureml.export>` exports the model as a 
-  directory with the dependencies necessary to deploy the model on Azure ML.
+.. rubric:: Deployment example (Python API):
 
-* :py:func:`deploy <mlflow.azureml.deploy>` deploys the model directly to Azure ML.
-  You need to set up your environment to work with the Azure ML CLI. You can do this by
-  starting a shell from the Azure ML Workbench application. You also have to set up all accounts
-  required to run and deploy on Azure ML. Where the model is deployed is dependent on your
-  active Azure ML environment. If the active environment is set up for local deployment, the model
-  is deployed locally in a Docker container (Docker is required).
+.. code:: python
 
-.. rubric:: Model export example
+    import mlflow.azureml
 
-.. code:: bash
+    from azureml.core import Workspace
+    from azureml.core.webservice import AciWebservice, Webservice
 
-    mlflow azureml export -m <path-to-model> -o test-output
-    tree test-output
+    azure_workspace = Workspace.get("<workspace-name>")
+    # Build an Azure ML container image for deployment
+    azure_image, azure_model = mlflow.azureml.build_image(model_path="<path-to-model>",
+                                                          workspace=azure_workspace,
+                                                          description="Wine regression model 1",
+                                                          synchronous=True)
 
-::
-  
-    test-output
-    ├── create_service.sh  - use this script to upload the model to Azure ML
-    ├── score.py - main module required by Azure ML
-    └── test-output - directory containing MLflow model in Python Function flavor
+    # Deploy the container image to ACI
+    webservice_deployment_config = AciWebservice.deploy_configuration()
+    webservice = Webservice.deploy_from_image(
+                        image=image, workspace=workspace, name="<deployment-name>")
+    webservice.wait_for_deployment()
 
-.. rubric:: Example workflow using the MLflow CLI
+    # After the image deployment completes, requests can be posted via HTTP to the new ACI
+    # webservice's scoring URI
+    print("Scoring URI is: %s", webservice.scoring_uri)
```

Comment:
Mind giving a simple curl example w/the wine dataset mentioned? (Maybe say we'll run from the examples/python wine thingy)